src/main.cpp

```diff
@@ -69,7 +70,9 @@ int main(int argc, char **argv) {
 #ifdef _WIN32
 	SetCurrentProcessExplicitAppUserModelID(MAKE_WSTR(PROGRAM_AUTHOR) L"." MAKE_WSTR(PROGRAM_NAME));
 #endif
+	Fl::keyboard_screen_scaling(0);
 	Fl::visual(FL_DOUBLE | FL_RGB);
+	fl_contrast_level(50);

```

Comment:
FLTK has a newer improved contrast algorithm, and a new default contrast level of 39 instead of 50 (which is a good thing). But we set it back to 50 anyway because it works best for all 12 themes, particularly in the menubar.